src/BackedEnum.php

```diff
@@ -28,17 +28,17 @@ public static function fromName(string $fqn, string $keyName): \BackedEnum
      */
     public static function tryFromName(string $fqn, string $keyName): ?\BackedEnum
     {
-        if (is_a($fqn, \BackedEnum::class, true) === false) {
-            throw new InvalidArgumentException('It is only possible to get names of backedEnums, "' . $fqn . '" provided');
+        if (!is_a($fqn, \BackedEnum::class, true)) {
+            throw new InvalidArgumentException(sprintf('It is only possible to get names of backedEnums, "%s" provided', $fqn));
         }
 
-        if (!defined("$fqn::$keyName")) {
+        if (!defined("{$fqn}::{$keyName}")) {
```

Comment:
Is this a code style or optimization I'm not aware of or just a personal preference?

Reply:
- curly braces make it safer
- easier to spot bugs
- an finally: use `sprintf` 😉